integration/test/basic/basic_test.go

```diff
@@ -63,25 +59,57 @@ func TestChartConfigChartsInstalled(t *testing.T) {
 		return
 	}
 
-	guestNamespace := "giantswarm"
 	guestG8sClient := g.G8sClient()
 
 	// Wait for chart configs as they may not have been created yet.
 	err := waitForChartConfigs(guestG8sClient)
 	if err != nil {
 		t.Fatalf("could not get chartconfigs %v", err)
 	}
+}
+
+func TestChartConfigPatchDeploy(t *testing.T) {
+	// These versions have no chartconfigs so we return early.
+	clusterOperatorVersion := os.Getenv("CLOP_VERSION_BUNDLE_VERSION")
```

Comment:
Why are we using `os.Getenv` everywhere here? If we need to read from the env multiple times, then this should be happening in a `env` package similar to how we do it in e.g. aws-operator.

Reply:
Yes, am aware of this. But didn't want to include the whole vendor e2e-restructuring in this PR also, otherwise it would blow up. Can provide this in a later PR.

Reply:
Moving in smaller steps with e2e is a good thing I think. Lets do a follow up PR.